pkg/api/transform_network.go

```diff
@@ -18,31 +18,31 @@
 package api
 
 type TransformNetwork struct {
-	Rules          NetworkTransformRules `yaml:"rules" doc:"list of transform rules, each includes:"`
-	KubeConfigPath string                `yaml:"kubeconfigpath" doc:"path to kubeconfig file (optional)"`
-	ServicesFile   string                `yaml:"servicesfile" doc:"path to services file (optional, default: /etc/services)"`
-	ProtocolsFile  string                `yaml:"protocolsfile" doc:"path to protocols file (optional, default: /etc/protocols)"`
+	Rules          NetworkTransformRules `yaml:"rules" json:"rules" doc:"list of transform rules, each includes:"`
+	KubeConfigPath string                `yaml:"kubeConfigPath,omitempty" json:"kubeConfigPath,omitempty" doc:"path to kubeconfig file (optional)"`
+	ServicesFile   string                `yaml:"servicesFile,omitempty" json:"servicesFile,omitempty" doc:"path to services file (optional, default: /etc/services)"`
+	ProtocolsFile  string                `yaml:"protocolsFile,omitempty" json:"protocolsFile,omitempty" doc:"path to protocols file (optional, default: /etc/protocols)"`
 }
 
 type TransformNetworkOperationEnum struct {
-	ConnTracking  string `yaml:"conn_tracking" doc:"set output field to value of parameters field only for new flows by matching template in input field"`
-	AddRegExIf    string `yaml:"add_regex_if" doc:"add output field if input field satisfies regex pattern from parameters field"`
-	AddIf         string `yaml:"add_if" doc:"add output field if input field satisfies criteria from parameters field"`
-	AddSubnet     string `yaml:"add_subnet" doc:"add output subnet field from input field and prefix length from parameters field"`
-	AddLocation   string `yaml:"add_location" doc:"add output location fields from input"`
-	AddService    string `yaml:"add_service" doc:"add output network service field from input port and parameters protocol field"`
-	AddKubernetes string `yaml:"add_kubernetes" doc:"add output kubernetes fields from input"`
+	ConnTracking  string `yaml:"conn_tracking" json:"conn_tracking" doc:"set output field to value of parameters field only for new flows by matching template in input field"`
+	AddRegExIf    string `yaml:"add_regex_if" json:"add_regex_if" doc:"add output field if input field satisfies regex pattern from parameters field"`
+	AddIf         string `yaml:"add_if" json:"add_if" doc:"add output field if input field satisfies criteria from parameters field"`
+	AddSubnet     string `yaml:"add_subnet" json:"add_subnet" doc:"add output subnet field from input field and prefix length from parameters field"`
+	AddLocation   string `yaml:"add_location" json:"add_location" doc:"add output location fields from input"`
+	AddService    string `yaml:"add_service" json:"add_service" doc:"add output network service field from input port and parameters protocol field"`
+	AddKubernetes string `yaml:"add_kubernetes" json:"add_kubernetes" doc:"add output kubernetes fields from input"`
```

Comment:
Elsewhere we use camelCase for the yaml and json definitions.

Reply:
same explanation here, it's because it's an enum, these names are values